canada_britishColumbia_ubc.md

```diff
@@ -7,5 +7,5 @@ years_published: 1980
 location: University of British Columbia, Vancouver, BC
 nosheets: 16
 infourl: http://resolve.library.ubc.ca/cgi-bin/catsearch?bid=12149901
-geojsonurl: https://github.com/ubc-lib-geo/spatial-indexes/blob/master/canada_britishColumbia_ubc.geojson
+geojsonurl: https://ubc-lib-geo.github.io/spatial-indexes/north-america/canada_britishColumbia_ubc.geojson
```

Comment:
The file URL was changed in the last commit, so it's fixed on develop already. This change can be safely removed.